ztoc/ztoc_test.go

```diff
@@ -444,6 +444,79 @@ func testZtocGeneration(t *testing.T, compressionAlgo string, generator tarGener
 	}
 }
 
+func BenchmarkZtocGeneration(b *testing.B) {
+	for _, tc := range testZtocs {
+		b.Run(tc.name, func(b *testing.B) {
```

Comment:
As a sanity check, `b.Run` is synchronous, correct? If not, we need `tc := tc` here too.

Reply:
I think so. This is how the official announcement of subtests runs sub-benchmarks:

https://go.dev/blog/subtests